install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -124,6 +124,11 @@ type WorkspaceConfig struct {
 	} `json:"contentService"`
 
 	EnableProtectedSecrets *bool `json:"enableProtectedSecrets"`
+
+	ImageBuilderMk3 struct {
+		BaseImageRepositoryName      string `json:"baseImageRepositoryName"`
+		WorkspaceImageRepositoryName string `json:"workspaceImageRepositoryName"`
+	} `json:"ImageBuilderMk3"`
```

Comment:
To match deploy-gitpod.sh, and match case for other struct names:
```suggestion
	} `json:"imageBuilderMk3"`
```